### PR TITLE
Kernel config lib

### PR DESCRIPTION
**Pull request by mwasilew**
This PR fixes issue with running IMA test with test-runner on a remote target. It contains the commit submitted as #321.

### CODE REVIEW

**Comment by mwasilew** [comment]
@roxell @chase-qi please take a look

File: automated/utils/test-runner.py [hunk 1]
```diff
@@ -907,6 +909,42 @@ def send_to_lava(self, data):
...
         subprocess.call(shlex.split(cmd))
 
+    def send_to_fiotest(self):
+        """
+        This method saves results as filesystem tree. This is required by
+        fiotest: https://github.com/foundriesio/fiotest/
+        """
+        # check if TEST_DIR variable is set
+        test_path = os.environ.get("TEST_DIR")
+        if not test_path:
```
Comment (chase-qi) [inline automated/utils/test-runner.py:919]: @mwasilew do you think it make sense to return here directly if test_path not set? When the path is not set(in most cases), printing the following two warning lines for every test probably is too much.

```
2021-11-23 14:54:24,216 - RUNNER.ResultParser: WARNING: TEST_DIR is not set
2021-11-23 14:54:24,216 - RUNNER.ResultParser: WARNING: NOT reporting result to fiotest
```

Comment (mwasilew) [reply automated/utils/test-runner.py:919]: Fine by me, I'll update the patch - change to debug

**Comment by mwasilew** [comment]
Looks like the other patch was approved. I'll rebase and resubmit.

**Comment by chase-qi** [comment]
Thanks, Milosz. It looks good me @roxell

**Pull request merged**